pkg/container-disk/container-disk.go

```diff
@@ -126,14 +126,14 @@ func GenerateContainers(vmi *v1.VirtualMachineInstance, podVolumeName string, bi
 			if vmi.IsCPUDedicated() || vmi.WantsToHaveQOSGuaranteed() {
 				resources.Limits = make(kubev1.ResourceList)
 				resources.Limits[kubev1.ResourceCPU] = resource.MustParse("10m")
-				resources.Limits[kubev1.ResourceMemory] = resource.MustParse("40M")
+				resources.Limits[kubev1.ResourceMemory] = resource.MustParse("5M")
```

Comment:
We have repeatedly OOM reports. Lets see if we can reduce that later (since it requires additional validation), by now just leaving it at `40M` and merge this earlier.